aws/src/test/java/org/apache/iceberg/aws/glue/GlueCatalogTest.java

```diff
@@ -70,7 +70,8 @@
   public void before() {
     glue = Mockito.mock(GlueClient.class);
     glueCatalog = new GlueCatalog();
-    glueCatalog.initialize(CATALOG_NAME, WAREHOUSE_PATH, new AwsProperties(), glue, null);
+    glueCatalog.initialize(CATALOG_NAME, WAREHOUSE_PATH, new AwsProperties(), glue,
+        LockManagers.defaultLockManager(), null);
```

Comment:
Since this is getting large, we may want to separate it into the lock manager PR that Glue uses, and a new PR for the dynamo lock manager.

Reply:
Sure, I can do that.